### PR TITLE
hsimple: pass Vc include dir as ROOT_INCLUDE_PATH

### PR DESCRIPTION
**Pull request by Axel-Naumann**
See https://github.com/root-project/root/issues/9594

### CODE REVIEW

**Review by vgvassilev — APPROVED** [review]

File: tutorials/CMakeLists.txt [hunk 1]
```diff
@@ -500,7 +500,12 @@ file(GLOB multithreaded RELATIVE ${CMAKE_CURRENT_SOURCE_DIR} ${multithreaded})
 #---Loop over all tutorials and define the corresponding test---------
 
 #---Define the primordial tutorials-----------------------------------
+if(Vc_INCLUDE_DIR)
+  set(HSIMPLE_ROOT_INCPATH "ROOT_INCLUDE_PATH=${Vc_INCLUDE_DIR}:${ROOT_INCLUDE_PATH}")
+endif()
+
 ROOT_ADD_TEST(tutorial-hsimple COMMAND ${ROOT_root_CMD} -b -l -n -q ${CMAKE_CURRENT_SOURCE_DIR}/hsimple.C
+    ENVIRONMENT ${HSIMPLE_ROOT_INCPATH}
```
Comment (bellenot) [inline tutorials/CMakeLists.txt:508]:
```suggestion
              ENVIRONMENT ${HSIMPLE_ROOT_INCPATH}
```

**Review by bellenot — APPROVED** [review]
I suggested indentation, otherwise LGTM, if it works 😉

**Comment by andresailer** [comment]
I am testing the branch in the lcgcmake system.

**Comment by andresailer** [comment]
@Axel-Naumann Could you rebase your branch to root master? I think you are missing the "breaking" commits.

**Comment by andresailer** [comment]
Our builds break at
```
[100%] Generating lib/modules.idx
[snip]
make[6]: *** [lib/modules.idx] Error 139
make[5]: *** [CMakeFiles/modules_idx.dir/all] Error 2
make[4]: *** [all] Error 2
```
I think this comes from https://github.com/root-project/root/blob/5d69eca85a2be137684807446878f782e4a368eb/CMakeLists.txt#L508-L519 which isn't part of the branch in this PR, but after a rebase to the ROOT master branch.

**Comment by andresailer** [comment]
I added two more ROOT_INCLUDE_PATH settings https://github.com/andresailer/root/commit/6584cc721e0925a6cdb0f1ef313e24f61639d113 after the rebase. Then we can build with lcgcmake without setting ROOT_INCLUDE_PATH on the outside.

**Comment by Axel-Naumann** [comment]
@vgvassilev see https://github.com/root-project/root/pull/9765#issuecomment-1026900510

That `[snip]` doesn't have any backtrace?

**Comment by Axel-Naumann** [comment]
@andresailer I think your commit is all we need - I forgot that we had the target in the main `CMakeLists.txt`!.

I'll close this PR; could you open a new one just with your commit, please?

**Pull request closed without merging**